test/manual/kerberos.test.js

```diff
@@ -97,6 +97,37 @@ describe('Kerberos', function () {
     });
   });
 
+  context('when passsing SERVICE_HOST', function () {
```

Comment:
was this supposed to be "when parsing"?

Reply:
Changed to "when passing SERVICE_HOST as an auth mech option"